de.metas.procurement.base/src/main/java/de/metas/procurement/base/order/async/PMM_GenerateOrders.java

```diff
@@ -68,19 +68,6 @@ public Result processWorkPackage(final I_C_Queue_WorkPackage workPackage, final
 		return Services.get(IQueueDAO.class).retrieveItemsSkipMissing(workpackage, I_PMM_PurchaseCandidate.class, ITrx.TRXNAME_ThreadInherited);
 	}
 
-	private List<I_PMM_PurchaseCandidate> getItemsWithQty(List<I_PMM_PurchaseCandidate> candidates)
```

Comment:
alright, good you removed it again; `OrderLineAggregation` already does that job (i saw it only now 😊 ).